windows/RNGestureHandler/RNGestureHandler/RNGestureHandlerModule.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "pch.h"
+#include "NativeModules.h"
+#include "RNGestureRegistry.h"
+
+using namespace winrt::Microsoft::ReactNative;
+
+#ifdef RNW61
+#define JSVALUEOBJECTPARAMETER
+#else
+#define JSVALUEOBJECTPARAMETER const &
+#endif
+
+namespace winrt::RNGestureHandler
+{
+    enum class RNGestureHandlerDirection
+    {
+        Right = 1,
+        Left = 2,
+        Up = 4,
+        Down = 8,
+    };

```

Comment:
I think this should belong in `RNGestureHandler.h`.

Reply:
Left it here to not introduce needless cyclic header dependency.